File: Niveau 1/43 - Bagarre générale/program.c
#include <stdio.h>

int main()
{
   int arignon, evaran;
   scanf("%d %d", &arignon, &evaran);
   
   int resultat = arignon - evaran;
   if (resultat > 10)
   {
      printf("La famille Arignon a un champ trop grand");
   }
   if (resultat < -10)
   {
      printf("La famille Evaran a un champ trop grand");
   }

   return 0;
}

File: Niveau 1/43 - Bagarre générale/program.py
superficieArignon = int(input())
superficieEvaran = int(input())
resultat = superficieArignon - superficieEvaran
if resultat < 0:
   resultat = - resultat
if resultat > 10:
   if superficieArignon < superficieEvaran:
      print("La famille Evaran a un champ trop grand")
   if superficieArignon > superficieEvaran:
      print("La famille Arignon a un champ trop grand")
